src/FSharp.Core/prim-types.fs

```diff
@@ -180,7 +180,7 @@ namespace Microsoft.FSharp.Core
         inherit Attribute()
         member _.CompiledName = compiledName
 
-    [<AttributeUsage (AttributeTargets.Struct ||| AttributeTargets.ReturnValue, AllowMultiple=false)>]  
```

Comment:
This would be a breaking change for anyone who is pinning FSharp.Core, right?

Reply:
I think it’s ok as we are extending the`AttributeTargets` so it will work on  old and new Fsharp.Core versions ?

Reply:
I meant if someone is using new compiler and old FSharp.Core (let's say 7.0.200), what would be the behaviour here?

Reply:
As far I can see the behaviour should be:
- New Compiler, old  FSharp.Core -> You can use `[<Struct>]` only on structs
- New Compiler, new FSharp.Core -> You can use `[<Struct>` on structs and classes
- Old Compiler, old  FSharp.Core -> You can use `[<Struct>]` only on structs
- Old Compiler, new  FSharp.Core -> You can use `[<Struct>]`  on structs and classes

Does this answer your question ?